src/App.tsx

```diff
@@ -280,83 +280,39 @@ const Screens = (t: TFunction) => {
         title: t('viewNames:positiveResult'),
         ...standardProps
       }
+    },
+    {
+      name: 'checker.intro',
+      component: CheckInIntro,
+      options: {
+        title: t('viewNames:symptomchecker'),
+        ...standardProps
+      }
+    },
+    {
+      name: 'checker.symptoms.1',
+      component: CheckInSymptoms,
+      options: {
+        title: t('viewNames:symptomchecker'),
+        ...standardProps
+      }
+    },
+    {
+      name: 'checker.final',
+      component: CheckInFinal,
+      options: {
+        title: t('viewNames:symptomchecker'),
+        ...standardProps,
+        headerLeft: null
+      }
     }
   ];
 };
 
-const SymptomsStack = () => {
-  const app = useApplication();
-  const {t} = useTranslation();
-  const {getNextScreen} = useSymptomChecker();
-
-  const initialRouteName = app.checks.length
-    ? ScreenNames.History
-    : getNextScreen();
-
-  return (
-    <Stack.Navigator
-      screenOptions={{
-        animationEnabled: true
-      }}
-      initialRouteName={initialRouteName}
-      headerMode="none">
-      <Stack.Screen
-        name={ScreenNames.History}
-        component={SymptomsHistory}
-        options={{
-          title: t('viewNames:symptomchecker'),
-          cardStyle: {
-            backgroundColor: colors.background
-          }
-        }}
-      />
-      <Stack.Screen
-        name="checker.consent"
```

Comment:
There's no longer a route named `'checker.consent'` with these changes, but there are still references to `'checker.consent'` in the code